static/js/dismiss_banner.js

```diff
@@ -0,0 +1,40 @@
+$(document).ready(function() {
+  function setCookie(name, value, days) {
+    let expires = "";
+    if (days) {
+      let date = new Date();
+      date.setTime(date.getTime() + days * 24 * 60 * 60 * 1000);

```

Comment:
Can we make this a Hugo param? It'd be nice to do that.

Reply:
I actually experimented with it a little, Setting a timestamp via hugo param can be a problem (`now` records the time from which server was live, and does not update it for later instances; could be a problem in production)

Reply:
Ah sorry. I don't mean a _timestamp_ as a Hugo param, I meant that the duration of suppression would be.

Reply:
That is done ;) I'll be pushing the commit soon